legacy/setup-opensuse.sh

```
@@ -1,15 +1,35 @@
+      fftw-devel findutils flex gcc-c++ gcc-fortran gdbm-devel gettext-devel git gperf gsl-devel \
+      gzip help2man hostname htop hwloc hwloc-devel joe m4 make libbsd-devel libtool libicu-devel \
+      glu-devel grpc-devel libunistring-devel libuuid-devel libX11-devel libXau-devel \
+      libXdmcp-devel libXext-devel libXfont-devel libXft-devel libxml2-devel \
+      libXmu-devel libXpm-devel libXrender-devel liblz4-devel libzstd-devel lsb-release Mesa-devel \
+      libXmu-devel libXpm-devel libXrender-devel liblz4-devel libzstd-devel lsb-release Mesa-devel \
+      ncurses-devel openssl-devel patch procps protobuf-devel python3 python3-devel \
+      readline-devel rsync sed subversion tar libtbb2 tbb-devel unzip wget which \
+      libxerces-c-devel xz xz-devel yaml-cpp-devel
 zypper clean
+
+os_version=${cat /etc/os-release | grep 'VERSION=' | cut -d\" -f2}
```

Shellcheck (0.11.0)

[warning] os_version appears unused. Verify use (or export if used externally).

(SC2034)